trnsysGUI/userSettings.py

```diff
@@ -0,0 +1,17 @@
+import pathlib as _pl
+from PyQt5.QtCore import QSettings  # pylint: disable=invalid-name
+
+
+class UserSettings:
+
+    _SETTINGS = QSettings("SPF", "pytrnsys")
```

Comment:
I would like to know more about what is in here.

Reply:
deprecated